setup.py

```diff
@@ -82,6 +82,7 @@
         "llm-blender>=0.0.2",
     ],
     "peft": ["peft>=0.8.0"],
+    "liger": ["liger-kernel-nightly==0.1.1.dev20240828063745"],
```

Comment:
I am not a big fan of pinning this dev version so we should update to the real package ASAP·

Reply:
we will cut a release in few hours